src/pages/workspace/accounting/qbo/import/QuickbooksClassesPage.tsx

```diff
@@ -1,77 +1,61 @@
-                        </View>
-                        <OfflineWithFeedback pendingAction={pendingFields?.syncClasses}>
-                            <View style={[styles.flex1, styles.alignItemsEnd, styles.pl3]}>
-                                <Switch
-                                    accessibilityLabel={translate('workspace.qbo.classes')}
-                                    isOn={isSwitchOn}
-                                    onToggle={() =>
-                                        Connections.updatePolicyConnectionConfig(
-                                            policyID,
-                                            CONST.POLICY.CONNECTIONS.NAME.QBO,
-                                            CONST.QUICK_BOOKS_CONFIG.SYNC_CLASSES,
-                                            isSwitchOn ? CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE : CONST.INTEGRATION_ENTITY_MAP_TYPES.TAG,
-                                        )
-                                    }
-                                />
-                            </View>
-                        </OfflineWithFeedback>
-                    </View>
-                    {isSwitchOn && (
-                        <OfflineWithFeedback>
-                            <MenuItemWithTopDescription
-                                interactive={false}
-                                title={isReportFieldsSelected ? translate('workspace.common.reportFields') : translate('workspace.common.tags')}
-                                description={translate('workspace.common.displayedAs')}
-                                wrapperStyle={styles.sectionMenuItemTopDescription}
-                            />
-                        </OfflineWithFeedback>
-                    )}
-                </ScrollView>
-            </ScreenWrapper>
-        </AccessOrNotFoundWrapper>
+            <ToggleSettingOptionRow
+                title={translate('workspace.accounting.import')}
+                switchAccessibilityLabel={translate('workspace.qbo.classes')}
+                isActive={isSwitchOn}
+                onToggle={() =>
+                    Connections.updatePolicyConnectionConfig(
+                        policyID,
+                        CONST.POLICY.CONNECTIONS.NAME.QBO,
+                        CONST.QUICKBOOKS_CONFIG.SYNC_CLASSES,
+                        isSwitchOn ? CONST.INTEGRATION_ENTITY_MAP_TYPES.NONE : CONST.INTEGRATION_ENTITY_MAP_TYPES.TAG,
+                    )
+                }
+                subMenuItems={
+                    <MenuItemWithTopDescription
+                        interactive={false}
+                        title={isReportFieldsSelected ? translate('workspace.common.reportFields') : translate('workspace.common.tags')}
+                        description={translate('workspace.common.displayedAs')}
+                        wrapperStyle={[styles.sectionMenuItemTopDescription, styles.mt4]}
+                    />
+                }
```

Comment:
I'd go with the previous approach because now the error for the toggle will be displayed under this menu item